asteroid/data/librimix_dataset.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pandas as pd
+import soundfile as sf
+import torch
+from torch.utils.data.dataset import Dataset
+import random as random
+
+
+class LibriMix(Dataset):
+    """" Dataset class for Librimix source separation tasks.
+
+    Args:
+        metadata_path (str): The path to the metatdata file
+        sample_rate (int) : The sample rate of the sources and mixtures
+        n_src (int) : The number of sources in the mixture
+        segment (int) : The desired sources and mixtures length in s
+    """
+
+    def __init__(self, metadata_path, sample_rate=16000,
+                 n_src=2, segment='full'):
```

Comment:
Can we keep the same behaviour as the other loaders : `segment=None` means full utterances

Reply:
Ok, but I had a problem with that when I tried to set `segment =None' in the conf file when you load the .yml  None becomes a str "None" and the dataloader breaks. How do you think we should deal with that ?

Reply:
I think `null` is what you're looking for.